on_premises/mlflow-server

```diff
@@ -0,0 +1 @@
+mlflow_user:$apr1$gbrjM.qB$f.PUdnqI6RodTXnpXAiFO/
```

Comment:
What is this file for?

Reply:
Oops it's not needed, I will remove it later.

Reply:
Thank you for your comments! I'm sorry to make you check basic things.
I made changes accordingly. I hope there's no more issues remaining...
Thanks again!!